snappi/snappicommon.py

```diff
@@ -143,13 +143,20 @@ def parent(self):
     def _get_property(self, name, default_value=None, parent=None, choice=None):
         if name not in self._properties or self._properties[name] is None:
             if isinstance(default_value, type) is True:
+                default_value._CHECK_REQUIRED = False
                 self._properties[name] = default_value(parent=parent, choice=choice)
             else:
                 self._properties[name] = default_value
```

Comment:
this part needs to be handled for default values of primitive types. e.g. when executing following
```python
import snappi
api = snappi.api()
config = api.config()
f1 = config.flows.flow(name='f1')[-1]
f1.size.fixed
print(config.serialize())
```
I'm getting following output:

```
{
  "flows": [
    {
      "name": "f1",
      "size": {
        "fixed": null
      }
    }
  ]
}
```

Although, `f1.size.increment` works fine (since when a type is passed as default_value, we initialize the type which in turn assigns defaults).